src/lib.rs

```diff
@@ -11,47 +11,73 @@ pub enum Stream {
 /// variables, is known to support hyperlink rendering.
 pub fn supports_hyperlinks() -> bool {
     // Hyperlinks can be forced through this env var.
-    if let Ok(arg) = std::env::var("FORCE_HYPERLINK") {
-        return arg.trim() != "0";
```

Comment:
Why is trimming required here? If someone sets the variable to something like `"0 "`, then it is their own fault. We don't trim the values of other variables and this shouldn't be an exception.

Reply:
changing this is a breaking change. Do you have measurements that point out a significant impact of having this `.trim()`?